campari/campari_runner.py

```diff
@@ -1,9 +1,11 @@
 import pathlib
-import galsim
+
+
 import numpy as np
 import pandas as pd
 
 # Astronomy
+import galsim
```

Comment:
I alphabetize regardless of whether the line starts with `import` or `from`.  Are you intentionally following a different convention?